eng/native/configurecompiler.cmake

```diff
@@ -797,4 +797,15 @@ elseif (NOT CLR_CMAKE_HOST_BROWSER)
 
     enable_language(ASM)
 
+    # The following block is a workaround for a bug in CMake with Clang 14. CMake
```

Comment:
https://github.com/dotnet/runtime/pull/75710/files#diff-36ef498ca978a03472b1216f6e3c603e6e3950d27bc7f70972c078b444f5e8b9R791-R796 seems to be working around the same problem. Is the existing workaround not effective? Do we need to have both of the workarounds?

Reply:
You have pointed to this PR, I guess you wanted to share a different link.

Reply:
The link is pointing 10 lines back in this very file, but the browser does not seem to be highlighting it correctly.

Reply:
Ah, sorry for missing that. That's interesting, it was not in my branch when I was making this change. I've then rebased to main before commit and haven't noticed there is that one. I guess I can close this PR then :-)

Reply:
Yup, line 796 is fixing the same thing.